aten/src/ATen/native/Collectives.cpp

```diff
@@ -24,7 +24,7 @@ at::Tensor all_gather_into_tensor(at::Tensor const& shard, const c10::string_vie
     TORCH_INTERNAL_ASSERT(false);
 }
 
-at::Tensor reduce_scatter_tensor(at::Tensor const& input, const c10::string_view reduceOp, int64_t scatter_dim, const c10::string_view tag, c10::ArrayRef<int64_t> ranks, int64_t group_size) {
+at::Tensor reduce_scatter_tensor(at::Tensor const& input, const c10::string_view reduceOp, const c10::string_view tag, c10::ArrayRef<int64_t> ranks, int64_t group_size) {
```

Comment:
curious why we delete the scatter_dim? I think it's still valuable to do the transpose thing in the  func collective level? We can do it in func collective python level

Reply:
reverting this to reduce the CI churn